modules/claims_api/app/sidekiq/claims_api/report_hourly_unsuccessful_submissions.rb

```diff
@@ -59,7 +59,7 @@ def notify(claims, va_claims, poa, itf, ews, from, to, env)
 
     def errored_submissions_exist?
       [@errored_claims, @va_gov_errored_claims, @errored_poa, @errored_itf, @errored_ews].any? do |var|
```

Comment:
don't know if we need the internal logic here. any would report if any exist. count positive is essentially duplicating that concept.